.circleci/config.yml

```diff
@@ -296,6 +297,15 @@ jobs:
               python scripts/hab2_bench/plot_bench.py
               # Assert the SPS number are up to standard
               python scripts/hab2_bench/assert_bench.py
+      - run:
+          name: Run HITL basic functionality
+          no_output_timeout: 30m
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat; cd habitat-lab
+              export PYTHONPATH=.:$PYTHONPATH
+              export MULTI_PROC_OFFSET=0 && export MAGNUM_LOG=quiet && export HABITAT_SIM_LOG=quiet
+              python test/hitl/test_hitl_main.py -s
```

Comment:
We shouldn't need to run this file explicitly. We should use `pytest` like the other tests in this file.